keyboards/phdesign/phac/readme.md

```diff
@@ -0,0 +1,26 @@
+# phac
+
+![phac](https://i.imgur.com/bQdYGMf.png)
+
+A rhythm game controller with 7 keys and 2 encoders on it.
+
+* Keyboard Maintainer: [Xufeng Tao](https://github.com/nonameCCC)
+* Hardware Supported: RP2040 minimal system
+* Hardware Availability: https://m.tb.cn/h.gLFXLaX?tk=7DpL3TuPVxx(currently not available outside China Mainland, as we could not afford the international express fee)
```

Comment:
Without the space after the link, the `(currently` is getting appended to the URL. 
Also, "currently no available outside of China" (or similar) is sufficient. Additional reasons don't need to be mentioned (part of the reason for the links is for hardware verification and potential acquisition for users)
```suggestion
* Hardware Availability: https://m.tb.cn/h.gLFXLaX?tk=7DpL3TuPVxx (currently not available outside of mainland China)
```